spring/src/main/java/uk/ac/bbsrc/tgac/miso/spring/ajax/LibraryControllerHelperService.java

```diff
@@ -364,6 +364,27 @@ public JSONObject changeLibraryLocation(HttpSession session, JSONObject json) {
 
     return JSONUtils.SimpleJSONResponse("Note saved successfully");
   }
+  
+  public JSONObject changeLibraryIdBarcode(HttpSession session, JSONObject json) {
+    Long libraryId = json.getLong("libraryId");
+    String idBarcode = json.getString("identificationBarcode");
+    
+    try {
+      if (!"".equals(idBarcode)) {
```

Comment:
also just found:
`miso.core.src.main.java.uk.ac.bbsrc.tgac.miso.core.util.LimsUtils.isStringEmptyOrNull(String s)`

Which seems like a very clear way of doing it - assuming we want null checking here.